### ISSUE TITLE
Some question

### ISSUE THREAD

**Issue by L1NINE** [Senwang98/MonoSKD#4]
Thank you for your hard work!

Also, I have some questions about the code.

1. The pretrained models of teacher and student are used as pretrain_model in the configs file? May I ask how you obtained these two weights?Could I use dataset to obtain the teacher&student model myself？

2. Why is the config file specifically divided into distill and DID modes? What is the purpose of doing this? Does DID means the DID-M3D model?

**Comment by Senwang98** [comment]
@L1NINE 
Thank you for your attention.
Re1: Yes, we load pretrained checkpoint in https://github.com/Senwang98/MonoSKD/blob/6764e9ddc1fc67b93cc2e32529a7649cc75822c1/configs/monodistill.yaml#L45. We  have provided teacher and student checkpoint link in README. You can use config/rgb_baseline.yaml to train your own teacher and student model. You can coment this line to train teacher model.https://github.com/Senwang98/MonoSKD/blob/6764e9ddc1fc67b93cc2e32529a7649cc75822c1/lib/models/DID.py#L106

Re2: **distill** is designed for knowledge distillation training, while **DID** is just single detector training.

**Comment by L1NINE** [comment]
Thank you for your explanation!

Based on the method you taught me, I changed the following code in DID py:
![dd](https://github.com/Senwang98/MonoSKD/assets/149569268/5c020c31-ba2d-4442-bb77-3999cc7fa0e9)
and I start training teacher model with the rgb_baseline.yaml. 
finally I got my teacher model, but I notice that my teacher model has some different with the model you provide.
The file size of the teacher model I trained is 246,220KB, when the model size you provide is 83,594.
Meanwhile I found I can't use pth_transfer.py to cut the model trained with DID type. 
Whether will it influence the finally end?
